arch/s390/dfltcc_detail.h

```diff
@@ -46,16 +46,16 @@ typedef enum {
 #define DFLTCC_FACILITY 151
 
 static inline dfltcc_cc dfltcc(int fn, void *param,
-                               unsigned char **op1, size_t *len1, const unsigned char **op2, size_t *len2, void *hist) {
+                               unsigned char **op1, size_t *len1, unsigned char **op2, size_t *len2, void *hist) {
```

Comment:
Don't remove `const` completely, replace it with `z_const` as that makes the code work with and without zlib compatibility mode.

Reply:
You're right, `next_in` is not `unsigned char *`, it's `z_const unsigned char *`. I'll amend the commit. Thanks!